eng/Publishing.props

```diff
@@ -13,6 +13,10 @@
     <!-- Include our "loose" PDBs when publishing symbols. -->
     <FilesToPublishToSymbolServer Include="$(ArtifactsDir)\symbols\**\*.pdb" />
```

Comment:
Is this entirely empty in this branch? We aren't doing anything particular to not publish PDBs.

Reply:
The root cause is not building the Microsoft.Web.Xdt.Extensions project. That's the only project we (a) need symbols for (to avoid an incomplete set of symbols) and (b) don't package (because the DLL ships in another package).